packages/conveyer/src/Conveyer.ts

```diff
@@ -312,10 +316,25 @@ class Conveyer extends Component<ConveyerEvents> {
   public updateItems() {
     const scrollAreaElement = this._scrollAreaElement;
     const itemSelector = this._options.itemSelector;
+    const resizeObserver = this._resizeObserver;
+    
     const itemElements = [].slice.call(
       itemSelector ? scrollAreaElement.querySelectorAll(itemSelector) : scrollAreaElement.children,
-    );
-    this.setItems(itemElements.map((el) => this._getItem(el)));
+      );
+      this.setItems(itemElements.map((el) => this._getItem(el)));
+      
+      if (resizeObserver){
+      const prevItemElements = this._itemElements;
```

Comment:
There will be no need to create itemElements.

Use `items.map(item => item.element)`.